### DISCUSSION TITLE
gnomock k3s error - Unable to get Kubeconfig from the container

### DISCUSSION THREAD

**Discussion by jianyongniu** [orlangure/gnomock#1025] [Q&A]
Error details below
    enl_agent_test.go:58: 
                Error Trace:    /home/jianyong/gitlab/ep5g/cm-agent-enl/testft/enl_agent_test.go:58
                Error:          Received unexpected error:
                                can't connect to container: canceled after error: failed to get kubeconfig: can't get kubeconfig bytes: kubeconfig unavailable: Get "http://127.0.0.1:32776/kubeconfig.yaml": read tcp 127.0.0.1:48272->127.0.0.1:32776: read: connection reset by peer

Any idea?

**Comment by orlangure** [comment]
Hi @jianyongniu, I would need a snippet that I can run to reproduce this. There are plenty of reasons for failing connection. Please include your environment info as well.
